### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by tumainimaganiko**
In this project I did the following:

- Delete all the boilerplate from Create React App (text, images, styles).
- Create a new directory called components.
- Inside components, I created a Calculator.js file.
- In Calculator.js, I created a React component that matches the provided design.
- Use functional components.
- Create at least 1 child component
- I imported Calculator.js and used it in my top-level component i.e App.js

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @tumainimaganiko, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has coming soon written in the key features, kindly know that an actual key feature(s) are needed to be written there.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by tumainimaganiko** [comment]
Hello @Olamarx 👋 
I thank you for your suggestions and I'm pretty sure that I have solved the requested issues.

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+function Span({ text, color }) {
+  return (
+    <span className={color}>{text}</span>
+  );
+}
+
+Span.propTypes = {
+  text: PropTypes.string.isRequired,
+  color: PropTypes.string.isRequired,
+};
```
Comment (TedLivist) [inline src/components/Calculator.js]: - [ ] **[OPTIONAL]** Kindly move this child component to its own file. As is the convention in React each component should be in its own file.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types';
+
+export default function Calculator() {
+  return (
+    <div className="container">
+      <Span text="0" color="display" />
+      <Span text="AC" color="white" />
+      <Span text="+/-" color="white" />
+      <Span text="%-" color="white" />
```
Comment (TedLivist) [inline src/components/Calculator.js]: - [ ] You have a typo error here. This should simply be "%", not "%-"

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @tumainimaganiko,

Good job so far!
✔️ Use of functional components
✔️ Implementation of child component
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tumainimaganiko** [comment]
Hello @TedLivist, Hopefully, I've met all the requirements

**Review by wayungi — APPROVED** [review]
Hi @tumainimaganiko , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] All linter checks are passing ✔️ 
- [ ]  Correct Gitflow ✔️ 
- [ ] All requested changes are implemented ✔️  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**